explore/src/main/scala/explore/programs/ProgramConfigRequestsTile.scala

```diff
@@ -0,0 +1,233 @@
+          ctx.pageUrl(AppTab.Observations, props.programId, Focused.singleObs(obsId))
+
+        def goToObs(obsId: Observation.Id): Callback =
+          ctx.pushPage(AppTab.Observations, props.programId, Focused.singleObs(obsId))
+
+        List(
+          rowColumn(ConfigRequestIdColumnId, _.request.id).setSize(90.toPx).sortable,
+          rowColumn(TargetColumnId, _.targetName).setSize(150.toPx).sortable,
+          configurationColumn(RAColumnId, _.refererenceCoordinates.ra)
+            .setCell(c => MathValidators.truncatedRA.reverseGet(c.value))
+            .setSize(110.toPx)
+            .sortable,
+          configurationColumn(DecColumnId, _.refererenceCoordinates.dec)
+            .setCell(c => MathValidators.truncatedDec.reverseGet(c.value))
+            .setSize(110.toPx)
+            .sortable,
+          configurationColumn(InstrumentColumnId, _.observingMode.tpe.instrument.shortName)
+            .setSize(110.toPx)
+            .sortable,
+          configurationColumn(FPUColumnId, _.observingMode.fpu).setSize(110.toPx).sortable,
+          configurationColumn(DisperserColumnId, _.observingMode.disperser)
+            .setSize(110.toPx)
+            .sortable,
+          configurationColumn(ImageQualityColumnId, _.conditions.imageQuality)
+            .setCell(_.value.label)
+            .setSize(80.toPx)
+            .sortable,
+          configurationColumn(CloudExtinctionColumnId, _.conditions.cloudExtinction)
+            .setCell(_.value.label)
+            .setSize(80.toPx)
+            .sortable,
+          configurationColumn(SkyBackgroundColumnId, _.conditions.skyBackground)
+            .setCell(_.value.label)
+            .setSize(80.toPx)
+            .sortable,
+          configurationColumn(WaterVaporColumnId, _.conditions.waterVapor)
+            .setCell(_.value.label)
+            .setSize(80.toPx)
+            .sortable,
+          rowColumn(ObservationsColumnId, _.obsIds)
+            .setCell(c =>
+              <.span(
+                c.value.toList
+                  .map(obsId =>
+                    <.a(
+                      ^.href := obsUrl(obsId),
+                      ^.onClick ==> (e =>
+                        e.preventDefaultCB >> e.stopPropagationCB >>
+                          goToObs(obsId)
+                      ),
+                      obsId.show
+                    )
+                  )
+                  .mkReactFragment(", ")
+              )
+            )
+            .setSize(150.toPx)
+            .setEnableSorting(false),
+          rowColumn(StatusColumnId, _.request.status)
+            .setCell(c => stateIcon(c.value))
+            .setSize(80.toPx)
+            .sortable
+        )
+    .useMemoBy((props, _, _) => // Rows
+      (props.configRequests, props.obs4ConfigRequests, props.targets)
+    ): (_, _, _) =>
+      (requests, obsMap, targets) => requests.map((_, r) => Row(r, obsMap, targets)).toList
+    // TODO: Save state
```

Comment:
I have another table to add. When I do that, I'll save the state of both of those, as well as the validations table, so I only have to do the UserPrefs updates once. I'll also need to add another tile.